components/merino_client/src/client.rs

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use serde_derive::*;
+
+use crate::error::{MerinoClientError, Result};
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct MerinoClientOptions {
```

Comment:
This is the minimal implementation of `merino_client.udl`! I found it helpful to have the UDL file opened side-by-side with this Rust code as I worked. 😊